From 7745250784a84dbe5f668f398608072112c1bcba Mon Sep 17 00:00:00 2001
From: Xiao Chen <xiao@cloudera.com>
Date: Tue, 21 Nov 2017 22:33:46 -0800
Subject: [PATCH 2695/2748] CLOUDERA-BUILD. CDH-62060. Revert "CLOUDERA-BUILD.
 Temporarily disable pre-commit and post-commit
 jobs until we figure out the dist-test timeouts."

This reverts commit 3ddf084f4b4d403830a2fe4854911146f5eea828.

Change-Id: If38e62dd6fe67739a58cc8527d566584f24808c9
---
 cloudera/test-distributed.sh |   16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/cloudera/test-distributed.sh b/cloudera/test-distributed.sh
index 549cb3d..7a62d93 100755
--- a/cloudera/test-distributed.sh
+++ b/cloudera/test-distributed.sh
@@ -47,14 +47,14 @@ EOF
 # Invoke grind to run tests
 grind -c ${DIR}/$SCRIPTS/env/grind.cfg config
 grind -c ${DIR}/$SCRIPTS/env/grind.cfg pconfig
-#grind -c ${DIR}/$SCRIPTS/env/grind.cfg test --artifacts -r 3 \
-#    -e TestContainerAllocation \
-#    -e TestJobHistoryEventHandler \
-#    -e TestSystemMetricsPublisher \
-#    -e TestContainerManagerSecurity \
-#    -e TestMRIntermediateDataEncryption \
-#    -e TestClientRMTokens \
-#    -e TestAMAuthorization
+grind -c ${DIR}/$SCRIPTS/env/grind.cfg test --artifacts -r 3 \
+    -e TestContainerAllocation \
+    -e TestJobHistoryEventHandler \
+    -e TestSystemMetricsPublisher \
+    -e TestContainerManagerSecurity \
+    -e TestMRIntermediateDataEncryption \
+    -e TestClientRMTokens \
+    -e TestAMAuthorization
     # TestClientRMTokens and TestAMAuthorization to be fixed in 5.8 (CDH-39590)
     # TestContinuousScheduling has been failing consistently, to be fixed in 5.8 (CDH-38830)
 
-- 
1.7.9.5

